ipn/ipnlocal/node_backend.go

```diff
@@ -379,6 +381,12 @@ func (nb *nodeBackend) SetNetMap(nm *netmap.NetworkMap) {
 	nb.netMap = nm
 	nb.updateNodeByAddrLocked()
 	nb.updatePeersLocked()
+	nv := magicsock.NodeViewsUpdate{}
+	if nm != nil {
+		nv.SelfNode = nm.SelfNode
+		nv.Peers = nm.Peers
+	}
+	nb.nodeViewsPub.Publish(nv)
```

Comment:
I assume this was intentional that we publish an empty update even if `nm == nil`, but it wasn't obvious to me why we'd want to do that. Maybe leave a comment? (Or possibly this ought to be a doc on the method itself, if `nm == nil` has a meaning of art).